roles/calibre-web/defaults/main.yml

```diff
@@ -0,0 +1,26 @@
+# The values here are defaults.
+# To override them edit /etc/iiab/local_vars.yml
+
+# Installation Variables
+calibreweb_install: False
+calibreweb_enabled: False
+
+# calibre-web folder to store its data and configuration files.
+calibreweb_home: "{{ content_base }}/calibre-web"
+
+calibreweb_port: 8083
+calibreweb_url: /calibre-web
```

Comment:
I'd be extremely tempted to make this `/books` (instead of "/calibre-web") to help kids/teachers/families remember it!

Reply:
True, calibre and calibre-web are rather separate.

But can we consider offering calibre-web at http://box/books if we believe it stands a strong chance of being the best interface for students and teachers both?

Reply:
@holta This is very young project, I think we need to test it properly before exposing it on top level box/books menus.